src/SWP/Bundle/ContentBundle/Processor/EmbeddedImageProcessor.php

```diff
@@ -113,6 +114,20 @@ public function applyByline(ArticleMediaInterface $articleMedia): string
         return $articleMedia->getByLine();
     }
 
+    private function appendImageCopyrightNotice(ArticleMediaInterface $articleMedia, \DOMElement $figCaptionNode): void
+    {
+        $element = new \DOMElement('span');
+        $figCaptionNode->appendChild($element);
+
+        $authorDiv = $figCaptionNode->childNodes[2];
+        $authorDiv->textContent = $this->applyCopyrightNotice($articleMedia);
+    }
+
+    public function applyCopyrightNotice(ArticleMediaInterface $articleMedia): string
```

Comment:
I see in travis this error: 

```
Behat\Testwork\Call\Exception\FatalThrowableError: Type error: Return value of SWP\Bundle\ContentBundle\Processor\EmbeddedImageProcessor::applyCopyrightNotice() must be of the type string, null returned in src/SWP/Bundle/ContentBundle/Processor/EmbeddedImageProcessor.php:128
```

You have to change the return type to `?string` here.